platform/engineering/design-docs/2021-06-11-small-batch-publishing.md

```diff
@@ -0,0 +1,353 @@
+| Rewrite Drupal pages                                      | X              |                            |                         |                                                                                                                                                         |
+| Create Drupal debug page                                  | X              |                            |                         |                                                                                                                                                         |
+| Download Drupal assets                                    | X              | X                          |                         | Drupal Assets will be pushed directly to S3.                                                                                                            |
+| Download application assets                               | X              |                            |                         |                                                                                                                                                         |
+| Create sitemap                                            | X              |                            |                         | Might move to new Lambda Job.                                                                                                                           |
+| Update robots.txt                                         | X              |                            |                         | Might move to new Lambda Job.                                                                                                                           |
+| Check for CMS URLs                                        | X              |                            |                         |                                                                                                                                                         |
+| Parse a virtual DOM from every .html file                 | X              | X                          | X                       |                                                                                                                                                         |
+| Ignore assets for build                                   | X              |                            |                         |                                                                                                                                                         |
+| Rewrite AWS URLs                                          |                | X                          | X                       |                                                                                                                                                         |
+
+
+
+#### Publish Batch API
```

Comment:
The implementation here might shift a bit to a more generic Event Bus using SNS and SQS.  I have experience with RabbitMQ and it looks like the SNS/SQS AWS pattern most closely matches RabbitMQ functionality: https://stackoverflow.com/questions/46880229/migrate-from-amqp-to-amazon-sns-sqs-need-to-understand-concepts

Reply:
I've updated the architecture diagram and add a note to this section regarding SNS and SQS patter.